app/views/course/assessment/question/multiple_responses/_option_fields.html.slim

```diff
@@ -1,6 +1,10 @@
 = content_tag_for(:tr, f.object, class: 'nested-fields') do
+  td.reorder.handle
+    span = fa_icon 'reorder'.freeze
   td = f.input :correct, label: false
   td = f.input :option, label: false, input_html: { class: ['multiple-response-option', 'airmode'] }
-  td = f.input :explanation, label: false, placeholder: t('.explanation_hint'),
+  td
+    = f.input :explanation, label: false, placeholder: t('.explanation_hint'),
     input_html: { class: ['multiple-response-explanation', 'airmode'] }
   td = link_to_remove_association t('.remove'), f
+  td.weight = f.input :weight, as: :integer, label: false
```

Comment:
Hide the field using js is unnecessary in this case, user need js to work with `cocoon` anyway.

And we are going to enforce javascript also, cause the frontend is in React.